packages/components/view-switcher/README.md

```diff
@@ -0,0 +1,109 @@
+# ViewSwitcher
+
+## Description
+
+ViewSwitchers allow users to toggle between two or more different views of the same, similar or related content items within the same space on screen.
+
+## When to use
+
+ViewSwitchers are frequently used to let users toggle between different formatting's, like with a grid view and a table view.
+
+They are also often used to narrow content groups.
+
+For example, use a content switcher if you have a list, such as “Import-results” and you want to divide it into views such as “All”, “Failed”, and “Success”.
```

Comment:
Personally I find difficult to differentiate between this example and when to use tabs.
Perhaps it's better to remove it.

Reply:
True. I agree with you.

Reply:
This has been updated in this commit: c7f430ef5530845eb75c43525902691340a711ef